app/javascript/packs/stylesheets/custom_table.scss

```diff
@@ -54,3 +54,21 @@
 .table-smaller-font {
   font-size: 0.9em;
 }
+
+.patients-table .table-hover tbody tr:hover {
+  background-color: #e4e4e4 !important;
```

Comment:
I don't _think_ these `!important` tags are needed

Reply:
also, thoughts on making this `background-color: #ddd` or  `background-color: #dedede` (based off what the darker colored rows are)

Reply:
I've removed the `!important` tags where they're not required. I've left the required ones.